### PR TITLE
RCAT-684 | Enable D11 jobs

### PR DESCRIPTION
**Pull request by Raksha-Bharuka**
Motivation
Enables jobs to test D11 readiness RCAT-684

Proposed changes
Following jobs are added for D11 with php8.3 -

- ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
- INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
- ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER
- INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER

### CODE REVIEW

File: .github/workflows/orca.yml [hunk 1]
```diff
@@ -61,18 +61,18 @@ jobs:
...
-          #   php-version: "8.1"
-          #   coveralls-enable: "FALSE"
-          # - orca-job: ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER
-          #   php-version: "8.1"
-          #   coveralls-enable: "FALSE"
-          # - orca-job: INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER
-          #   php-version: "8.1"
-          #   coveralls-enable: "FALSE"
+
+          # Testing Drupal 11 in php 8.3.
```
Comment (TravisCarden) [inline .github/workflows/orca.yml:65]: This adds no value.

```suggestion
```

File: .github/workflows/orca.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ on:
         -   cron: "0 0 * * 0"
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```
Comment (TravisCarden) [inline .github/workflows/orca.yml:14]: What's the reason for this?

File: .github/workflows/orca.yml [hunk 3]
```diff
@@ -61,18 +61,18 @@ jobs:
...
-          # - orca-job: INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER
-          #   php-version: "8.1"
-          #   coveralls-enable: "FALSE"
+
+          # Testing Drupal 11 in php 8.3.
+          - orca-job: ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
+            php-version: "8.3"
+          - orca-job: INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
+            php-version: "8.3"
+            -enable: "FALSE"
```
Comment (TravisCarden) [inline .github/workflows/orca.yml:70]: It looks like this used to be `coveralls-enable` and got cut off. I notice the other instances of `coveralls-enable` have been removed. I assume they were all supposed to have been kept intact.

**Review by TravisCarden — CHANGES_REQUESTED** [review]

File: .github/workflows/orca.yml (hunk 3 above)
Comment (Raksha-Bharuka) [reply .github/workflows/orca.yml]: yes it was `coveralls-enable`, fixed it

File: .github/workflows/orca.yml (hunk 2 above)
Comment (secretsayan) [reply .github/workflows/orca.yml:14]: This is required for D11 jobs as it requires an updated version of SQLite which is only available with ubuntu-24.04.

Comment (danepowell) [reply .github/workflows/orca.yml:14]: TIL "latest" doesn't actually mean "latest" (`latest` is currently 22.04): https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Comment (TravisCarden) [reply .github/workflows/orca.yml:14]: Ah, the old `$x != $x` problem. 😅 Noted. Thanks, @danepowell. 😛

**Review by TravisCarden — APPROVED** [review]

**Pull request merged**